CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Change Log
 
+## [0.16.0] 7 November 2023
+
+### Added
+
+- Added relative node's priority, which determines their execution order
+  - Added API calls to set/get node's priority:
+    - `rgl_graph_node_set_priority`
+    - `rgl_graph_node_get_priority`
+
+### Changed
+
+- Changed the way RGL runs graphs
+  - It is performed fully asynchronous now
+- Moved Entity ID from SBT to OptixInstance
+  - Efficiency has been improved
+  - Maximum value has been reduced
+    - The Entity ID is now held in a 28-bit integer (OptiX limit) instead of a 32-bit integer
```

Comment:
```suggestion
- Changed internal implementation of Entity ID feature
  - The Entity ID is now held in a 28-bit integer (OptiX limit) instead of a 32-bit integer
```